### PR TITLE
fix: log

### LINKED ISSUE #4272: [BUG] New Log layout
image

```
Notice: Undefined index: debug in /var/www/FreshRSS/app/Models/Themes.php on line 114
```

And missing horizontal scrollbar.

@Frenzie WebSub issue on your site? :-) Or is it a bug in my FreshRSS instance?

_Originally posted by @Alkarex in https://github.com/FreshRSS/FreshRSS/issues/4204#issuecomment-1067360873_

### PR DESCRIPTION
**Pull request by math-GH**
Closes #4272 

Changes proposed in this pull request:

- icon for debug log level
- improve css
- scrollable table

How to test the feature manually:

1. see log page

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

File: app/Models/Themes.php [hunk 1]
```diff
@@ -78,6 +78,7 @@ public static function alt($name) {
 			'category' => '☷',
 			'close' => '❌',
 			'configure' => '⚙',
+			'debug' => '🐛',
```
Comment (Alkarex) [inline app/Models/Themes.php:81]: I like :-)

File: p/themes/Mapco/_components.css [hunk 2]
```diff
@@ -0,0 +1,289 @@
+/* stylelint-disable property-no-vendor-prefix */
```
Comment (Alkarex) [inline p/themes/Mapco/_components.css]: Where does this file come from? Maybe an error?

Comment (math-GH) [reply p/themes/Mapco/_components.css]: sorry. I will delete it.

**Comment by Alkarex** [comment]
Maybe include `log-debug` as well, or use another selector for the whole column?
https://github.com/FreshRSS/FreshRSS/blob/ae54a590b9962916dc1a62ba55e8d8e8ac0fc215/p/themes/base-theme/template.css#L1527-L1531

image

**Pull request merged**